conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conans import ConanFile, CMake
+
+
+class Libgit2Conan(ConanFile):
+    name = "libgit2"
+    version = "0.28.0"
+    license = "GPL v2"
+    url = "https://github.com/libgit2/libgit2"
+    description = """`libgit2` is a portable, pure C implementation of the Git core methods
+provided as a linkable library with a solid API, allowing to build Git
+functionality into your application."""
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
+
+    generators = "cmake"
+    # exports_sources = "CMakeLists.txt", "src/*", "include/*", "cmake/*", "deps/*", \
+    #     "tests/*", "libgit2.pc.in", "libgit2_clar.supp", "fuzzers/*"
+    exports_sources = "*"
```

Comment:
I'd expect that only sources in "src/*", "include/*" and "deps/*" are required

Reply:
I thought so too, but I've hit some issues with the testing tool `libgit2_clar` not being compiled, but then I remembered I forgot a Conan default that puts all binaries in a `bin` directory.
I need to discuss this with someone who knows libgit2 better than me.